keyboards/duck/octagon/v1/config.h

```diff
@@ -21,9 +21,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x444B // Duck ("DK")
 #define PRODUCT_ID      0x6050
```

Comment:
Any reason this isn't `0x3847` like the Octagon v2?

Reply:
The stars in the night illuminated a portion of the fifth galaxy. This sent tremors across the mortal planes affecting keyboards far and wide. 

tl;dr: I'm blind.